docs/scos/user/features/202108.0/promotions-discounts-feature-overview.md

```diff
@@ -107,7 +107,7 @@ When a cart is evaluated on a level that has a rule and a rule group, the rule g
 
 ### Discount threshold
 
-Threshold is a minimum number of items in cart that should fulfill all the specified decision rules for the discount to be applied.
+Threshold is a minimum number of items in cart that should fulfill all the specified decision rules for the discount to apply.
```

Comment:
```suggestion
Threshold is a minimum number of items in the cart that should fulfill all the specified decision rules for the discount to apply.
```

Reply:
If we have "the" before cart, then "that should fulfill" would refer to the cart, and not to the threshold, as intended.